app/views/sessions/mfa_prompt.html.erb

```diff
@@ -0,0 +1,7 @@
+<% @title = t('multifactor_authentication') %>
+
+<% if @user&.webauthn_enabled? %>
+  <%= render "sessions/webauthn_prompt" %>
+<% end %>
+
+<%= render "sessions/otp_prompt" %>
```

Comment:
Can we please this a horizontal split? vertical looks clunky. `l-half--l` and `l-half--r` may be helpful.